src/dex-helper/irequest-wrapper.ts

```diff
@@ -1,3 +1,45 @@
+export type Method =
```

Comment:
I think better just to import from `axios`

Reply:
Axios is devDependecy

Reply:
If someone want to use dex-lib without axios it should be able to do it I guess